pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -101,15 +101,15 @@
                                 3: {
                                     <% all_tags.forEach(function(tag) { %>
                                     "<%= tag.name %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= tag.name %>\b/.test(e);
+                                        return new RegExp('\\b<%= tag.name %>\\b').test(e);
```

Comment:
I think we should fully escape the `tag.name` for use in a regexp. This could be one solution: https://stackoverflow.com/a/3561711

Reply:
It has been escaped. I used the Mozilla recommended escape and added a '\' to their list of special characters.